scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SchedulerController.java

```diff
@@ -97,6 +97,11 @@ public String getLogo350Url() {
         return SchedulerImagesUnbundled.LOGO_350;
     }
 
+    @Override
+    public String getPortalLogo() {
+        return SchedulerImagesUnbundled.pps_run;
```

Comment:
Do you really need those methods that only return a public static field ? You could directly access to it

Reply:
Yes I need this because the value will be either PPS_RUN or PPS_Deploy